src/templates/project-details.js

```diff
@@ -30,31 +35,61 @@ const DetailsPage = ({ pageContext: { project } }) => {
           </aside>
         </article>
 
-        <article className={s.overview} className={s.content_container}>
-          <div className={s.overview_content} className={s.main}>
+        <article className={classnames(s.overview, s.content_container)}>
```

Comment:
⚠️ Please replace all underscore_case with camelCase throughout this PR

Reply:
Done